docs/index.md

```diff
@@ -0,0 +1,64 @@
+# Index
+
+-   [Blogs](https://github.com/spiffe/tornjak/blob/main/docs/blogs.md)
+
+-   [Debugging, Hints and Tips for Solving Common Problems with Tornjak](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-hints.md)
+    - [Tornjak Deployment](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-hints.md#tornjak-deployment)
+    - [Tornjak Configuration](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-hints.md#tornjak-configuration)
+    - [User Management](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-hints.md#user-management)
+
+-   [JSON Format for New Entry](https://github.com/spiffe/tornjak/blob/main/docs/newEntry-json-format.md)
+
+-   [Server plugin: Authentication "Keycloak"](https://github.com/spiffe/tornjak/blob/main/docs/plugin_server_authentication_keycloak.md)
+
+-   [Server plugin: Authorization "RBAC"](https://github.com/spiffe/tornjak/blob/main/docs/plugin_server_authorization_rbac.md)
+
+-   [Server plugin: Datastore "SQL"](https://github.com/spiffe/tornjak/blob/main/docs/plugin_server_datastore_sql.md)
+
+-   [Server plugin: SPIRECRDManager](https://github.com/spiffe/tornjak/blob/main/docs/plugin_server_spirecrd.md)
+
+-   [Tornjak Agent](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-agent.md)
+    - [Tornjak SPIRE Server Agent](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-agent.md#tornjak-spire-server-agent)
+    - [APIs](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-agent.md#apis)
+    - [Authentication](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-agent.md#authentication)
+    - [Authorization](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-agent.md#authorization)
+
+-   [Tornjak Manager](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-manager.md)
+    - [Global Visibility](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-manager.md#global-visibility)
+    - [Identity Policy Management](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-manager.md#identity-policy-management)
+    - [Auditability of Identities And Use For Operations/Forensics](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-manager.md#auditability-of-identities-and-use-for-operationsforensics)
+    - [Management of SPIRE Identity Configurations](https://github.com/spiffe/tornjak/blob/main/docs/tornjak-manager.md#management-of-spire-identity-configurations)
```

Comment:
I think these can be removed from the list - the documentation is old-ish but will be updated in the coming months